hugo/layouts/partials/extra/gtm_optimize_container.html

```diff
@@ -0,0 +1,6 @@
+{{ if eq (getenv "ENABLE_ANALYTICS_TRACKING") "true" }}
+<!-- Loads Google Optimize container to run experiments or personalizations -->
+<script>
+    ga('require', '{{ .Site.Params.google_optimize }}');
```

Comment:
Does it matter that this is in quotes? `'{{ .Site.Params.google_optimize }}'`

Reply:
Ya, otherwise it seems to be invalid js.

Reply:
Yayayay!